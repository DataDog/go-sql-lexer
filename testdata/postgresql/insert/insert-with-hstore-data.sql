INSERT INTO user_profiles (profile) VALUES ('"height"=>"2m", "weight"=>"70kg"');
