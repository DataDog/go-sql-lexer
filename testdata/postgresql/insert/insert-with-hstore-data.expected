[
    {
        "expected": "INSERT INTO user_profiles ( profile ) VALUES ( ? )",
        "statement_metadata": {
            "size": 19,
            "tables": [
                "user_profiles"
            ],
            "commands": [
                "INSERT"
            ],
            "comments": [],
            "procedures": []
        }
    }
]